.github/workflows/01-build-and-release.yml

```diff
@@ -0,0 +1,138 @@
+          rm djgpp-linux64-gcc1210.tar.bz2
+          popd
+      - name: Build project
+        run: |
+          cd $GITHUB_WORKSPACE/src
+          ls -lh main.c
+          . /opt/djgpp/setenv
+          make
+          ls -lh output/sbemu.exe
+      - name: Build FreeDOS SBEMU USB image
+        run: |
+          cd /tmp
+          wget https://www.ibiblio.org/pub/micro/pc-stuff/freedos/files/distributions/1.3/official/FD13-LiteUSB.zip
+          wget https://www.freedos.org/download/verify.txt
+          grep -q "64a934585087ccd91a18c55e20ee01f5f6762be712eeaa5f456be543778f9f7e  FD13-LiteUSB.zip" verify.txt
+          echo "64a934585087ccd91a18c55e20ee01f5f6762be712eeaa5f456be543778f9f7e  FD13-LiteUSB.zip" | shasum -a 256 --check
+          unzip FD13-LiteUSB.zip
+          rm FD13-LiteUSB.zip
+          wget https://github.com/Baron-von-Riedesel/Jemm/releases/download/v5.84pre2/JemmB_v584.zip
+          echo "719dfea1fdc609c5256fee4b8ad6db6880ff7a29f0ddc5dcaaa609a87771d218  JemmB_v584.zip" | shasum -a 256 --check
+          wget https://github.com/crazii/HX/releases/download/v0.1-beta3/HDPMI32i.zip
+          echo "4b7fc576d6c5dfd3fe7290fd1d0c0229118d01a55c8325d487452c2d6e0c3bbd  HDPMI32i.zip" | shasum -a 256 --check
+          wget https://www.ibiblio.org/pub/micro/pc-stuff/freedos/files/repositories/1.3/base/ctmouse.zip
+          echo "a891124cd5b13e8141778fcae718f3b2667b0a49727ce92b782ab11a8c4bb63a  ctmouse.zip" | shasum -a 256 --check
+          mkdir /tmp/mnt
+          sudo mount FD13LITE.img /tmp/mnt -t vfat -o loop,offset=$((63*512)),rw,uid=$(id -u),gid=$(id -g)
+          mkdir /tmp/mnt/sbemu
+          cp $GITHUB_WORKSPACE/src/output/sbemu.exe /tmp/mnt/sbemu
+          cd /tmp/mnt
+          mkdir jemm
+          cd jemm
+          unzip /tmp/JemmB_v584.zip
+          ls -lh JEMMEX.EXE
+          ls -lh JLOAD.EXE
+          ls -lh QPIEMU.DLL
+          cd ..
+          mkdir hdpmi
+          cd hdpmi
+          unzip /tmp/HDPMI32i.zip
+          ls -lh HDPMI32i.EXE
+          cd ..
+          mkdir ctmouse
+          unzip -j "/tmp/ctmouse.zip" "BIN/CTMOUSE.EXE" -d "ctmouse"
+          sed -i 's/DEVICE=\\FREEDOS\\BIN\\HIMEMX.EXE/DEVICE=\\JEMM\\JEMMEX.EXE MAXEXT=2097152\nDEVICE=\\JEMM\\JLOAD.EXE \\JEMM\\QPIEMU.DLL/g' fdconfig.sys
+          mv setup.bat setup.bak
+          echo 'LH \HDPMI\HDPMI32I.EXE' > setup.bat
+          echo 'LH \SBEMU\SBEMU.EXE' >> setup.bat
+          echo 'LH \CTMOUSE\CTMOUSE.EXE' >> setup.bat
+          cd /tmp
+          rm JemmB_v584.zip
+          rm HDPMI32i.zip
+          rm ctmouse.zip
+          sudo umount /tmp/mnt
+          mv FD13LITE.img SBEMU-FD13-USB.img
+          xz -k -9e SBEMU-FD13-USB.img
+          mv SBEMU-FD13-USB.img.xz $GITHUB_WORKSPACE/
+      - name: Generate release tag
+        id: tag
+        run: |
+          echo "release_tag=UserBuild_$(date +"%Y.%m.%d_%H-%M")" >> $GITHUB_OUTPUT
+      - name: Release FreeDOS SBEMU USB image
```

Comment:
While the FreeDOS image is appreciated, why not build a small self-contained release that can be put onto an existing DOS install? Having to download the FreeDOS image and extracting the SBEMU bits sounds tedious if one already has a DOS setup of some kind.

Reply:
I think we need both.
The single release is essential, and the FreeDOS image will be extra good that helps people build a USB bootdisk easily.

Reply:
Look, I'm not really in favor of putting abemu inside a pre-configured or pre-installed .img image, I think it's unnecessary, I think focusing on development would be much more important, if I'm useful, please add me as part. I tested, in FreeDOS it still occurs, some error related to hdpmi, a strange error causes the mouse to overload the memory. and the system crashes. In my NEDR-DOS, a variant of Open-DOS, this error does not occur, and it runs very well, I suggest you test it with the game Kings Quest V, you will see the error

Reply:
As a coder I agree with you, we need focus on the dev & bugfixes. But in a user's view a pre installed img is good to have, and we have nothing to lose. Leave this to volkertb and we just focus on the rest.